test/tests/misc_scoping.py

```diff
@@ -0,0 +1,199 @@
+# expected: fail
```

Comment:
Could we move the tests that work out of here and leave a comment as to what's not working?